src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -129,6 +132,43 @@ def add_parser_init_flow(subparsers):
     )
 
 
+def add_parser_save_flow(subparsers):
+    """Add flow save parser to the pf flow subparsers."""
+    epilog = """
+Examples:
+
+# Creating a flex flow folder in a specific path:
+pf flow save --path my-awesome-flow --entry intent:extract_intent --code src
```

Comment:
what will happen if `my-awesome-flow` is an absolute path and `code` is also passed?

Reply:
`code` will be copied to `my-awesome-flow`

Reply:
actually this is the original expected usage